File: CPP/beginner/1013.cpp
#include <stdio.h>

int main() {
 
   int A = 0, B = 0, C = 0, Maior = 0;
   
    scanf("%d %d %d", &A, &B, &C);
   
    Maior = A;
    
    if (B > Maior) {
        Maior = B;
    }
  
    if (C > Maior) {
        Maior = C;
    }
   
    printf("%d eh o maior\n", Maior);
   
    return 0;
}

File: CSharp/beginner/1013.cs
using System;

class Program
{
    static void Main(string[] args)
    {
        string i = Console.ReadLine();

        string[] nums = i.Split(' ');

        int a = int.Parse(nums[0]);
        int b = int.Parse(nums[1]);
        int c = int.Parse(nums[2]);

        int bigger = (a + b + Math.Abs(a - b)) / 2;
        bigger = (c + bigger + Math.Abs(c - bigger)) / 2;

        Console.WriteLine($"{bigger} eh o maior");
    }
}
